tensorflow_transform/tf_utils.py

```diff
@@ -176,6 +175,10 @@ def reduce_batch_count_mean_and_var(x, reduce_instance_dims):
       tf.sparse_reduce_sum if isinstance(x, tf.SparseTensor) else tf.reduce_sum)
   axis = None if reduce_instance_dims else 0
 
+  # If we have a dense tensor with nans, do not include in the sum
```

Comment:
Let's put this logic in reduce_sum_fn instead

Reply:
reduce_sum_fn is just an alias to tf.reduce_sum above, and this seems a bit long for a lambda. So I'll plan to add a `reduce_batch_sum` similar to `reduce_batch_count`. Let me know if you'd prefer another option